workloads/api-load/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -x
+
+source ./common.sh
+
+CR=api-load-crd.yaml
+
+log "###############################################"
+log "api-load tests: ${TESTS}"
+log "###############################################"
+
+run_workload ${CR}
+
```

Comment:
@venkataanil Can we add a removal of the benchmark operator at the end here? Something like this from the kube-burner one
```
remove_benchmark_operator ${OPERATOR_REPO} ${OPERATOR_BRANCH}
```

Reply:
Thanks Russ. Agree, I will update the PR with your suggestion.

Reply:
@dry923 I have gone through other workloads now and none of them are removing benchmark operator after the test is run. So lets follow the same methodology and skip calling remove_benchmark_operator.